src/HealthChecks.UI.Data/HealthChecksDb.cs

```diff
@@ -5,23 +5,21 @@ namespace HealthChecks.UI.Data
 {
     public class HealthChecksDb : DbContext
     {
-        public DbSet<HealthCheckConfiguration> Configurations { get; set; }
+        public DbSet<HealthCheckConfiguration> Configurations => Set<HealthCheckConfiguration>();
```

Comment:
Why is this done?

Reply:
Never mind, false positive. I will close the PR